src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -563,26 +563,38 @@ function packScalarsToVector(
 }
 
 /**
- * Generates a Case for the param and unary interval generator provided.
+ * Builds a Case for the param and unary interval generator provided.
  * The Case will use use an interval comparator for matching results.
  * @param param the param to pass into the unary operation
  * @param ops callbacks that implement generating an acceptance interval for a unary operation
  */
-export function makeUnaryToF32IntervalCase(param: number, ...ops: PointToInterval[]): Case {
+function makeUnaryToF32IntervalCase(param: number, ...ops: PointToInterval[]): Case {
   param = quantizeToF32(param);
 
   const intervals = ops.map(o => o(param));
   return { input: [f32(param)], expected: anyOf(...intervals) };
 }
 
 /**
- * Generates a Case for the params and binary interval generator provided.
+ *  @returns an array of Cases for a given operation over a range of inputs
```

Comment:
You say `given operation` here, but `ops` is an array. By reading this comment, it's not obvious to me what the function does if you pass multiple ops.

Reply:
Same thing for the functions below.